docs/integrations/notification/index.rst

```diff
@@ -0,0 +1,164 @@
+########################################
+ Monitoring Experiment Through Webhooks
+########################################
+
+Monitoring experiment status is a vital part of working with Determined. In order to integrate
+Determined into your existing workflows, you can make use of webhooks to update other systems,
+receive emails, slack messages, and more when an experiment is updated.
+
+*****************************
+ Security and Signed Payload
+*****************************
+
+Each Webhook request will include a signed payload that users can use to verify that webhook calls
+from Determined are genuine. This will require that users define a ``webhook_signing_key`` for
+signing. If a webhook signing key is not provided then one will be generated for the user.
+
+Setting the Key
+===============
+
+The signing key can be set in the following ways:
+
+#. In the master yaml configuration file by adding:
+
+.. code::
+
+   security:
+       webhook_signing_key: <signing_key>
+
+2. By setting a ``DET_WEBHOOK_SIGNING_KEY`` environment variable.
+#. By specifying a ``---security-webhook-signing-key`` flag.
+
+Retrieving the Key
+==================
+
+The key can be found in the cluster configuration. For example it will be returned in
+``api/v1/master/config``.
+
+Event Payload
+=============
+
+Currently we will support two separate types of Webhooks ``Slack`` and ``Default``. A payload for a
+``Default`` webhook will contain information about the event itself, the trigger for the event, and
+the appropriate entity that triggered the event. Since we only support experiment state changes, all
```

Comment:
I would say here that the shape of `event_data` is determined by `event_type`, that this is an example payload for `EXPERIMENT_STATE_CHANGE`, and that other types may be structured differently.